verify_rpms/rpm_verifier.py

```diff
@@ -211,6 +211,28 @@ def set_output_and_status(
     return output, failures_occurred
 
 
+def aggregate_results(processed_image_list: list[ProcessedImage]) -> dict[str, Any]:
+    """
+    Aggregate results from all images
+    :param processed_image_list: list of ProcessedImages
+    :return: dictionary with the aggregated results
+    """
+    aggregated_rpms_keys: list[str] = []
+    aggregated_unsigned_rpms: list[str] = []
+    aggregated_results: dict[str, Any] = {}
+
+    for image in processed_image_list:
+        if image.error != "":
+            aggregated_results["error"] = image.error
+            return aggregated_results
+        aggregated_rpms_keys = aggregated_rpms_keys + image.signed_rpms_keys
```

Comment:
you can write it like this (which is a bit cleaner):

```
aggregated_rpms_keys += image.signed_rpms_keys
```

Reply:
Done